src/fondue/Fondue.js

```diff
@@ -20,20 +20,37 @@ export default class Fondue {
 	}
 
 	get isColor() {
-		return this.colorFormats.length >= 1;
+		return this.colorFormats.length > 0;
+	}
+
+	get hasFeatures() {
+		return (
```

Comment:
Can there be other kind of features besides `fixed` or `optional`?

Reply:
No, only these two types. But with the refactor, this is no longer an issue anyway :-)